boot/bootdata/hivecls.inf

```diff
@@ -216,6 +216,12 @@ HKCR,"chm.file","",0x00000000,%CHMFILE%
 HKCR,"chm.file\DefaultIcon","",0x00020000,"%SystemRoot%\hh.exe,0"
 HKCR,"chm.file\shell\open\command","",0x00020000,"%SystemRoot%\hh.exe %1"
 
+; HTA Files
+HKCR,".hta","",0x00000000,"htafile"
+HKCR,"htafile","",0x00000000,"HTML Application"
+HKCR,"htafile\DefaultIcon","",0x00020000,"%SystemRoot%\system32\mshta.exe,-1"
+HKCR,"htafile\shell\open\command","",0x00020000,"%SystemRoot%\system32\mshta.exe %1 %*"
```

Comment:
Just checked, the %1 must be inside quotes, so you should fix this to:
```
"%SystemRoot%\system32\mshta.exe ""%1"" %*"
```
(please note the exact number of quotes around %1).

Reply:
Fixed in latest commit. Thanks!